sjb/config/common/test_cases/origin_built_installed_release.yml

```diff
@@ -44,11 +44,6 @@ extensions:
           sudo atomic pull --storage ostree docker:openshift/openvswitch:$( cat ./ORIGIN_TAG )
           sudo atomic pull --storage ostree docker:openshift/node:$( cat ./ORIGIN_TAG )
         fi
-    - type: "script"
-      title: "tag openshift/origin for system containers"
-      repository: "aos-cd-jobs"
-      script: |-
-        sudo docker tag openshift/origin:$( cat ./ORIGIN_TAG ) openshift/origin:$( cat ./ORIGIN_TAG ).$(cat ORIGIN_PKG_VERSION | grep -Po "^\-\w\.\w\.(\w)" | cut -d '.' -f 3) || true
```

Comment:
What about to keep it here? We will have to same problem once `origin_built_installed_release_310.yml` is forked. It would make more sense to copy-paste the `origin_built_installed_release.yml` -> `origin_built_installed_release_310.yml` and just remove the `|| true` part in the process. Plus adding comment here saying "Remove the `|| true` once forked into `origin_built_installed_release_3N.yml`".

@smarterclayton WDYT?